apps/google-analytics-4/lambda/src/app.spec.ts

```diff
@@ -79,4 +79,16 @@ describe('app', () => {
       expect(response.body[0]).to.have.property('propertySummaries');
     });
   });
+
+  describe('GET /sampleData/runReportResponse', () => {
+    it('responds with 200 for sample with views data', async () => {
+      const response = await chai.request(app).get('/sampleData/runReportResponseHasViews.json');
+      expect(response).to.have.status(200);
+    });
+
+    it('responds with 200 for sample with no views data', async () => {
+      const response = await chai.request(app).get('/sampleData/runReportResponseNoViews.json');
+      expect(response).to.have.status(200);
```

Comment:
This is definitely not this specific to this PR. Just want to note this for retro to call out a suggestion to use a http status code class so that we don't have to hard code http status numbers.

Something like https://www.npmjs.com/package/http-status or if there's a built-in expressJs library (not familiar with ExpressJs at all)

Reply:
Good idea!